js/views/userPageVw.js

```diff
@@ -307,6 +307,25 @@ module.exports = baseVw.extend({
           self.model.set({ownPage: self.options.ownPage});
           self.render();
           !self.currentItemHash && self.loadingDeferred.resolve();
+          
+          // Handle was requested
+          if (self.model.get('page').profile.handle) {
+            var profile = self.model.get('page').profile,
```

Comment:
One mild concern I have about this is that we are continually adding to localStorage here. While the LS limit is pretty high (at least relative to cookies), it's still finite and we wouldn't want to run out of space. I'm wondering if we should cap the list at some number - could be high (maybe 1000). After that, for any new handles, we should remove the oldest stored handle first to create space for the new one. I suppose this would require you to store a timestamp with each handle (which, humorously enough, would take addition space to store).

@jjeffryes What do you think? Valid concern, or am I overthinking? We definitely don't want to fill up local storage because we use it for more critical things than this.